concourse/scripts/s3init.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
The S3 changes have already been merged. Looks like this branch needs a rebase anyway, possibly to pick that up. Could you rebase and test whether it works?